src/Our.Umbraco.StackedContent/Web/Helpers/ViewHelper.cs

```diff
@@ -1,15 +1,36 @@
 ﻿using System.IO;
+using System.Linq;
 using System.Web;
 using System.Web.Mvc;
 using System.Web.Routing;
+using Umbraco.Core.Logging;
 
 namespace Our.Umbraco.StackedContent.Web.Helpers
 {
-    internal static class ViewHelper
+    public static class ViewHelper
     {
-        class DummyController : Controller { }
+        private class DummyController : Controller { }
 
-        public static string RenderPartial(string partialName, object model, string[] viewLocations)
+        private static readonly RazorViewEngine ViewEngine = new RazorViewEngine
```

Comment:
I'm wondering if there are any concerns about keeping a `RazorViewEngine` instance around in a static field?
Previously the code was creating a new instance per render.  I can't think of a problem with it, but felt that I should raise it (somehow).

Reply:
Viewengines are normally static, doesn't really make sense not to keep them static.
![image](https://user-images.githubusercontent.com/1074004/37854656-19536f42-2eec-11e8-8c24-c2127f01ceff.png)

Reply:
guess that's perhaps not 100% accurate, but they are accessed in a static/singleton kinda way .. 😅